jadler-jdk/src/main/java/net/jadler/stubbing/server/jdk/JdkStubHttpServer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2012 - 2015 Jadler contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package net.jadler.stubbing.server.jdk;
+
+import com.sun.net.httpserver.HttpServer;
+import net.jadler.RequestManager;
+import net.jadler.stubbing.server.StubHttpServer;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+/**
+ * Stub server implementation based on {@link HttpServer} which is part of JDK.
+ */
+public class JdkStubHttpServer implements StubHttpServer {
+    private final HttpServer server;
+
+    public JdkStubHttpServer(int port) {
+        try {
+            server = HttpServer.create(new InetSocketAddress(port), 100);
```

Comment:
Do you remember what is the reason for the `100` constant? What about using 0 to enforce the system default value?